fdbserver/workloads/MutationLogReaderCorrectness.actor.cpp

```diff
@@ -60,7 +60,7 @@ struct MutationLogReaderCorrectnessWorkload : TestWorkload {
 
 		beginVersion = deterministicRandom()->randomInt64(
 		    0, std::numeric_limits<int32_t>::max()); // intentionally not max of int64
-		records = deterministicRandom()->randomInt(0, 1e6);
+		records = deterministicRandom()->randomInt(0, 1e5);
```

Comment:
FYI: https://github.com/apple/foundationdb/pull/10542 set this to 500e3.

Reply:
I think there is no significant difference of the test coverage between 1e6 or 1e5 or 500e3. Set the records to a small value makes the test faster to complete, increasing the chance that the test gets tested.